sbt-plugin-test/build.sbt

```diff
@@ -32,7 +35,10 @@ lazy val withDOM = project.settings(baseSettings: _*).
     name := "Scala.js sbt test w/ DOM",
     jsDependencies ++= Seq(
         RuntimeDOM,
-        "org.webjars" % "jquery" % "1.10.2" / "jquery.js"
+        "org.webjars" % "jquery" % "1.10.2" / "jquery.js",
```

Comment:
Because we know it works. It was brand new when it put it there. The point here is more to test the `jsDependencies` mechanism than to test jQuery.